### PR TITLE
FIX issues with scope storage

### PR DESCRIPTION
**Pull request by TimoPtr**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+0.13.3-KOLIBREE
+------
+- Fix issue on Android 11 and save screenshots in Download directory
+
+0.13.2-KOLIBREE
```
Comment (lookashc) [inline CHANGELOG.md]: we're on 0.14

**Review by lookashc — COMMENTED** [review]
Did you try to run our Screenshot testing setup with this updated plugin sir? I would like to make sure it's working before we merge it.

File: CHANGELOG.md (hunk 1 above)
Comment (TimoPtr) [reply CHANGELOG.md]: FIXED

**Comment by TimoPtr** [comment]
Tested on our current setup it works like it should

**Pull request merged**